core/templates/pages/exploration-editor-page/translation-tab/translation-tab.component.spec.ts

```diff
@@ -86,6 +86,20 @@ describe('Translation tab component', () => {
     closeTour() {}
   }
 
+  class MockUserExplorationPermissionsService {
+    getPermissionsAsync() {
+      return Promise.resolve({
+        canVoiceover: true
+      } as ExplorationPermissions);
+    }
+
+    fetchPermissionsAsync() {
+      return Promise.resolve({
+        canVoiceover: true
+      } as ExplorationPermissions);
+    }
+  }
+
```

Comment:
Can we spy instead of mocking the service?

Reply:
i have tried to use spy on the place of mock the test starts to fail in that condition in  seed 198.